test_util/main.tf

```diff
@@ -35,7 +35,7 @@ variable "windowsagent_num" {
 
 variable "ssh_public_key_file" {
   type = "string"
-  default = "~/.ssh/id_rsa.pub"
+  default = "./tf-dcos-rsa.pem.pub"
```

Comment:
This is still wrong. It should point to a variable. Please wait for our reviews before merging.

Reply:
It's fixed in https://github.com/dcos/dcos/pull/6956/files#diff-bbea99011468909d2eb6c34caa9b3c33R59.

Reply:
@orsenthil @cgawron1 I agree what @jeschkies said.

Reply:
@DominikDary  - @jeschkies 

* The motivation was to   _avoid waiting on_ each other's reviews during the initial phase of the system, stabilize it with fixing known issues. (I will encourage you to do the same too). Once the system is stable, I agree that we should strive to keep the system stable and it makes sense for waiting.

* It is okay to fix it further in subsequent changes. LIke it is being done here: https://github.com/dcos/dcos/pull/6956/files#diff-bbea99011468909d2eb6c34caa9b3c33R59

Reply:
> The motivation was to avoid waiting on each other's reviews during the initial phase of the system, stabilize it with fixing known issues.

I feel rushing changes results in the opposite of moving fast. A change might be okay but we need to know about it.